modules/custom/inope_policy_review/inope_policy_review.module

```diff
@@ -0,0 +1,121 @@
+<?php
+/**
+ * @file
+ * The iNope Policy Review Module.
+ */
+
+/**
+ * Implements hook_menu().
+ */
+function inope_policy_review_menu() {
+  $items = array();
+
+  $items['node/%/review'] = array(
+    'title callback' => 'inope_policy_review_tab_title_callback',
+    'title arguments' => array(1),
+    'description' => 'Review of policy nodes.',
+    'page callback' => 'drupal_get_form',
+    'page arguments' => array('inope_policy_review_form'),
+    'access arguments' => array('view published content'),
+    'type' => MENU_LOCAL_TASK,
+  );
+
+  return $items;
+
+}
+
+/**
+ * The iNope Policy Review Page Title Callback.
+ */
+function inope_policy_review_tab_title_callback($node_nid) {
```

Comment:
for nids in drupal you can probably assume that node isn't necessary.

If I had to do this i'd just go with $nid, its pretty common across core, so feel free to duplicate the pattern.  If you have some kind of weird arg -- you might want to be more specific. $uid is probably another one that you can just use.